Bottom: 6fe77a3b133b7cfe991ea1b4f73e206c9e4d4b67
Top:    1f0fd239f6d1e30db99ffd2d6f9c37160c42c406
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 00:48:58 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index f351ef2..94402f4 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1298,6 +1298,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
        *
        */
 
+      fprintf (stderr, "Subtracting pin + via drills\n");
+      r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+      r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+
       if (info.poly == NULL)
         {
           fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
